src/wfdlgs3.c

```diff
@@ -788,6 +788,18 @@ FormatSelectDlgProc(register HWND hDlg, UINT wMsg, WPARAM wParam, LPARAM lParam)
                 // that parent, even if it is hidden.
                 ShowWindow(hDlg, SW_HIDE);
 
+                pszDlgTitle = NULL;
+                dwCchTitleLength = GetWindowTextLength(hDlg);
+                if (dwCchTitleLength > 0)
+                {
+                    dwCchTitleLength++;
+                    pszDlgTitle = malloc(ByteCountOf(dwCchTitleLength));
+                    if (pszDlgTitle != NULL)

```

Comment:
sorry; just noticed.  The WinFile standard for memory allocation is LocalAlloc().